site/docs/how-to-release.md

```diff
@@ -422,7 +422,7 @@ spark-runtime jar for the Spark installation):
 ```bash
 spark-shell \
     --conf spark.jars.repositories=${MAVEN_URL} \
-    --packages org.apache.iceberg:iceberg-spark3-runtime:{{ icebergVersion }} \
+    --packages org.apache.iceberg:iceberg-spark-runtime-${SPARK_VERSION}_${SCALA_VERSION}:{{ icebergVersion }} \
```

Comment:
where are these variables set? I don't see any places that would set spark/scala version

Reply:
I guess it's up to the user to populate `SPARK_VERSION` and `SCALA_VERSION`.

Reply:
Yes, it's up to the users and I've added some notes.

Reply:
I'd rather just set the variables like this, since this is a `bash` script:

```
# replace these with the correct versions
SPARK_VERSION=3.5
SCALA_VERSION=2.12
```

Reply:
You mean just provide some default versions? Do we need to verify all combinations of versions?

Reply:
I would just use the latest Spark version and default the scala version to 2.12